### PR TITLE
New version: CHIRPSoftware.CHIRP-next version 20241206

### PR DESCRIPTION
**Pull request by radoslew**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/198172)

### CODE REVIEW

File: manifests/c/CHIRPSoftware/CHIRP-next/20241206/CHIRPSoftware.CHIRP-next.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: CHIRPSoftware.CHIRP-next
+PackageVersion: '20241206'
+InstallerType: nullsoft
+ReleaseDate: 2024-12-06
+Installers:
+- Architecture: x64
+  InstallerUrl: https://archive.chirpmyradio.com/chirp_next/next-20241206/chirp-next-20241206-installer.exe
+  InstallerSha256: 5f84e57dd0ea99dac08aed16ec86876f00e70139
```
Comment (stephengillie) [inline manifests/c/CHIRPSoftware/CHIRP-next/20241206/CHIRPSoftware.CHIRP-next.installer.yaml:10]: > Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][0][InstallerSha256] Description: Failed to match regex specified by 'pattern' constraint. Error context: <root>[Installers][0] Description: Failed to validate against schema associated with property name 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #0 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: CHIRPSoftware.CHIRP-next.installer.yaml Manifest Error: Failed to process field. [InstallerSha256] Manifest Error: Required field missing. [InstallerSha256]

Hi @radoslew,

This hash seems too short.

Comment (radoslew) [reply manifests/c/CHIRPSoftware/CHIRP-next/20241206/CHIRPSoftware.CHIRP-next.installer.yaml]: Now it should be fine - sorry!

**Review by radoslew — COMMENTED** [review]
Hash corrected

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**